pkg/kubectl/resource_printer.go

```diff
@@ -940,9 +940,10 @@ func printCluster(c *federation.Cluster, w io.Writer, options PrintOptions) erro
 		statuses = append(statuses, "Unknown")
 	}
 
-	if _, err := fmt.Fprintf(w, "%s\t%s\t%s\n",
+	if _, err := fmt.Fprintf(w, "%s\t%s\t%s\t%s\n",
 		name,
 		strings.Join(statuses, ","),
+		"",
```

Comment:
Actually we should do the other way round - please remove VERSION column. It should have been removed in https://github.com/kubernetes/kubernetes/pull/28186.